default.yml

```diff
@@ -2,108 +2,100 @@
 # https://rubocop.readthedocs.io
 
 AllCops:
-  DisplayCopNames: true
   Exclude:
     - '*.gemspec'
     - 'bin/**/*'
     - 'data/**/*'
-    - 'db/schema.rb'
-    - 'db/data_schema.rb'
+    - 'db/**/*'
     - 'gemfiles/**/*'
-    - 'lib/tasks/circle_0.rake' # Ignore circleci code
+    - 'lib/tasks/circle_0.rake'
     - 'log/**/*'
+    - 'node_modules/**/*'
     - 'public/**/*'
     - 'tmp/**/*'
     - 'vendor/**/*'
+    - 'Guardfile'
```

Comment:
Does it break if we include this one? I think overall we exclude way too many files just to be conservative.

Reply:
No it doesn't break. Removing 👍